pkg/dockerregistry/server/auth.go

```diff
@@ -201,6 +196,11 @@ func TokenRealm(options map[string]interface{}) (*url.URL, error) {
 func newAccessController(options map[string]interface{}) (registryauth.AccessController, error) {
 	log.Info("Using Origin Auth handler")
 
+	ctx, ok := options["_context"].(context.Context)
```

Comment:
Ooops. Sorry.  I looked at `WithUserClient`, but was thinking about this change :)